.bazelignore

```diff
@@ -10,13 +10,19 @@ bazel-testlogs
 # treated as directories with valid BUILD files for the main repo.
 # Any directory with a WORKSPACE in it should be added here, with
 # an entry like `bazel-{workspacename}`
+examples/bzlmod/bazel-bin
 examples/bzlmod/bazel-bzlmod
+examples/bzlmod/bazel-out
+examples/bzlmod/bazel-testlogs
+examples/bzlmod/other_module/bazel-bin
```

Comment:
I would not expect the bazel symlinks to be included here. Or at leastthey should be included for all examples.

Reply:
Yeah, I've been confused by this. It's not very clear to me why the symlinks get picked up sometimes and not others. Sometimes adding to this ignore list helps, sometimes I have to delete the symlinks. It seems like some interaction between the outer bazel, deleted packages, the ignore file, and some sort of caching.